src/Neo/SmartContract/Native/WhitelistedFeeContract.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2015-2025 The Neo Project.
+//
+// WhitelistedFeeContract.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.VM;
+using Neo.VM.Types;
+
+namespace Neo.SmartContract.Native
+{
+    public class WhitelistedFeeContract : IInteroperable
```

Comment:
Will we have variable fee for those contracts? Or all will be free?
If so, let's think about a different name for this Native Contract

Reply:
It's a map with the method name / fixed execution fee